index.js

```diff
@@ -67,6 +70,26 @@ class Conf {
 	clear() {
 		this.store = obj();
 	}
+	onDidChange(key, callback) {
+		let currentValue = this.get(key);
+
+		const onChange = () => {
+			const oldValue = currentValue;
+			const newValue = this.get(key);
+
+			try {
+				assert.deepEqual(newValue, oldValue);
+			} catch (err) {
+				currentValue = newValue;
+				callback.call(this, newValue, oldValue);
+			}
+		};
+
+		this.events.on('change', onChange);
+
+		// Return a function which removes the listener.
```

Comment:
This comment is also just describing what is clear in the code, IMHO.

Reply:
I agree with you about all these comments too, but I was just erring on the side of clarity 👍

Reply:
I'll make it clear in the readme that it returns a function to unsubscribe, anyway.